GenderPayGap.WebUI/BackgroundJobs/Jobs/PurgeUsersJob.cs

```diff
@@ -76,7 +74,7 @@ private static async void PurgeUser(User user, DateTime pinExpiryDate, string ru
 
         private static void DeleteUserAndAuditLogs(User user)
         {
-            var dataRepository = Program.ContainerIOC.Resolve<IDataRepository>();
+            var dataRepository = MvcApplication.ContainerIoC.Resolve<IDataRepository>();

```

Comment:
I know this is to do with DI. But I still struggle to get my head around things like this. Are you able to talk me through it (again 🙄) at some point please?

Reply:
Sure - happy to have a chat at some point